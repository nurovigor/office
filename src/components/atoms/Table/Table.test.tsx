import React from 'react';
import { render } from '@testing-library/react';
import { Table } from './';

describe('Table component', () => {
	it('renders Preloader component', () => {
		const { asFragment } = render(<Table direction={'top'} isActive={false} />);
		expect(asFragment()).toMatchSnapshot();
	});
	it('renders with a className equal to the Active', () => {
		const { container } = render(<Table direction={'top'} isActive={true} />);
		// eslint-disable-next-line testing-library/no-node-access
		expect(container.firstChild).toHaveClass('Active');
	});
});
