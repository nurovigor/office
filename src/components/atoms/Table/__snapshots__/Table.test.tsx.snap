// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table component renders Preloader component 1`] = `
<DocumentFragment>
  <div
    class=""
    data-direction="top"
  />
</DocumentFragment>
`;
